base/util.jl

```diff
@@ -601,3 +601,39 @@ function runtests(tests = ["all"]; ncores::Int = ceil(Int, Sys.CPU_THREADS::Int
               "including error messages above and the output of versioninfo():\n$(read(buf, String))")
     end
 end
+
+# stat.jl helpers which require methods that are loaded later
+
+function getusername(uid::Union{UInt32, UInt64})
+    if Sys.islinux()
```

Comment:
These apps are simply wrappers around the POSIX.1 standard `getpwuid_r` and `getgrgid_r` functions, which we can call directly

Reply:
Implemented